Bluepill-cli/BPInstanceTests/Resource Files/save_retry.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="All tests" tests="0" failures="0" errors="0" time="0.000000">
+    <testsuite tests="0" failures="0" errors="0" time="0.937023" timestamp="2018-12-06T15:35:23GMT-08:00" name="BPSampleAppCrashingTests.xctest">
+        <testsuite tests="0" failures="0" errors="0" time="0.936033" timestamp="2018-12-06T15:35:23GMT-08:00" name="BPSampleAppCrashingTests">
```

Comment:
Aren't the counters wrong here (and above)? I would've expected:

```
<testsuite tests="2" failures="1" errors="0" ...
```

Reply:
Since I'm just re-using the code from a previous test with retry enabled and then checking to see if it makes that file I suspect this might be an issue with the JUnitReporter.